julia-mode.el

```diff
@@ -67,6 +67,35 @@ User can still use `abbrev-mode' or `expand-abbrev' to substitute
 unicode for LaTeX even if disabled."
   :type 'boolean)
 
+(defconst julia-mode--latexsubs-partials
+  (let ((table (make-hash-table :test 'equal)))
+    (maphash (lambda (latex subst)
```

Comment:
Unused function arguments should be prefixed with an underscore. Byte compiler complains about `subst`.